build-tools/heroku/tailwind/styles.css

```diff
@@ -353,11 +353,16 @@ code {
 /* Lists */
 
 ul {
-    list-style: disc outside none;
+  list-style: disc outside none;
 }
 
 ol {
-    list-style: decimal outside none;
+  list-style: decimal outside none;
+}
+
+.text-display-copy ul, .text-display-copy ol,
+.text-business-copy ul, .text-business-copy ol {
+  margin-left: 2em;
```

Comment:
I think we can achieve the same with `list-style-position` and when using Tailwind the `list-inside` class:

https://tailwindcss.com/docs/list-style-position

I would prefer this approach as it is less code and better aligns with our CSS already being as much Tailwind as possible. Of course, I the goal is to also give it a bit of an indent on the text itself, I agree with your solution.

Reply:
`list-style-position` gives a different effect for dangling lines though, and arguably one that's less scannable (or at least, in my opinion). Examples:

```
list-style-position: inside
```

<img width="362" alt="image" src="https://user-images.githubusercontent.com/524162/218559997-f5ac6b14-8f8c-4cae-bd72-e7ccc627ef96.png">

```
list-style-position: outside
```

<img width="347" alt="image" src="https://user-images.githubusercontent.com/524162/218560055-744e5cbf-9ffa-4adc-a3ef-282bc4505ec3.png">


We can't really control the bullet style on the lists we're interested in, as most of them are rendered from Markdown, and so we don't write the HTML directly where we would add the Tailwind `class="list-inside"` class.